### PR TITLE
Habit Tracker Widget - Refactor Dashboard, Journal and WeekScheduleBar, merge duplicated CSS

### LINKED ISSUE #199: Missing unique keys for lists in DraggableWidget.jsx and JournalHabitTracker.jsx
**Describe the bug**
A react warning regarding a lack of unique keys on list items is produced when running the GitHub CI tests.

**To Reproduce**
1. Make a PR
2. Let the tests run
3. See that the following errors occurs:

```
Warning: Each child in a list should have a unique "key" prop.
      
      Check the render method of Page. See https://reactjs.org/link/warning-keys for more information.
          at /home/runner/work/3Lancers/3Lancers/telos-frontend/src/dnd/DraggableWidget.jsx:20:11
```
and

```
 Warning: Each child in a list should have a unique "key" prop.
      
      Check the render method of `JournalHabitTracker`.
```
**Expected behavior**
No warning message for missing keys should appear

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Linux (Github CI)
 - Browser: N/A
 - Version: N/A

**Smartphone (please complete the following information):**
 - Device: N/A
 - OS: N/A
 - Browser N/A
 - Version N/A

**Additional context**
As these are warnings, they are not a high priority, however, do need attention.

### LINKED ISSUE #220: Habit Tracker Widget - Refactor Dashboard, Journal and WeekScheduleBar, merge duplicated CSS
**Is your feature request related to a problem? Please describe.**
As a code contributor, I want to avoid having duplicate code so that if functionality changes it only has to change it in one place, and to ensure features are identical across views without having to double-check

**Describe the solution you'd like**
Identical CSS in both Dashboard and Journal views such as AddIcon, WeekSchedule and menuContainer e.t.c.. moved into a single CSS module that can be imported by the Journal, Dashboard and WeekScheduleBar

### PR DESCRIPTION
**Pull request by ysway**
Closes #220 

## Proposed Changes
- Add comments to habit tracker
- Add a unique key for each child in a list for JournalHabitTracker.jsx #199 
- Refactor code
- Merge duplicated CSS file into a single CSS module

### CODE REVIEW

**Review by lhsunyc — APPROVED** [review]
Looks good

File: telos-frontend/src/components/habit-tracker/HabitTracker.module.css [hunk 1]
```diff
@@ -236,10 +237,10 @@
...
-    color: white !important;
-    padding: 0px !important;
+  color: white !important;
+  padding: 0px !important;
 }
 
 .addIconButtonSvg {
-    font-size: 2rem !important;
+  font-size: 2rem !important;
 }
```
Comment (milkyway404) [inline telos-frontend/src/components/habit-tracker/HabitTracker.module.css]: missing line

**Review by milkyway404 — APPROVED** [review]
Great work with the comprehensive comments! Just a small fix for newline as commented would be great. I've cloned and tested on local and it works the same as before~

**Review by yzho976 — APPROVED** [review]
Looks fine for me. Good work. These comments are all very nice and in detail too.

File: telos-frontend/src/components/habit-tracker/HabitTracker.module.css (hunk 1 above)
Comment (ysway) [reply telos-frontend/src/components/habit-tracker/HabitTracker.module.css]: added

**Pull request merged**